marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.tsx

```diff
@@ -166,6 +172,11 @@ const AdvancedSettingsDialog = (props) => {
         let response = await Axios.get(`/api/steps/${stepType}/${props.stepData.name}`);
 
         if (response.status === 200) {
+          if(stepType === 'ingestion' && response.data.stepDefinitionName !== 'default-ingestion'){
```

Comment:
Normally, I'd say the UI shouldn't be bound to this knowledge, and the middle tier should instead insert e.g. a "isCustomStepDefinition" property for the UI to be bound to. But since I think "Step Definition Name" should always be present and that we should eventually do that, I think it's fine for the UI to know about this detail for now.